### PR TITLE
Twitch.TwitchStudio version 0.104.7

### PR DESCRIPTION
**Pull request by sanyer**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/34777)

### CODE REVIEW

File: manifests/t/Twitch/TwitchStudio/0.104.7/Twitch.TwitchStudio.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Twitch.TwitchStudio
+PackageVersion: 0.104.7
```
Comment (jedieaston) [inline manifests/t/Twitch/TwitchStudio/0.104.7/Twitch.TwitchStudio.installer.yaml:5]:
```suggestion
PackageVersion: 8.0.0
```

File: manifests/t/Twitch/TwitchStudio/0.104.7/Twitch.TwitchStudio.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Twitch.TwitchStudio
+PackageVersion: 0.104.7
```
Comment (jedieaston) [inline manifests/t/Twitch/TwitchStudio/0.104.7/Twitch.TwitchStudio.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 8.0.0
```

File: manifests/t/Twitch/TwitchStudio/0.104.7/Twitch.TwitchStudio.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Twitch.TwitchStudio
+PackageVersion: 0.104.7
```
Comment (jedieaston) [inline manifests/t/Twitch/TwitchStudio/0.104.7/Twitch.TwitchStudio.yaml:5]:
```suggestion
PackageVersion: 8.0.0
```

**Review by jedieaston — CHANGES_REQUESTED** [review]
Don't forget to change the folder name too.

File: manifests/t/Twitch/TwitchStudio/0.104.7/Twitch.TwitchStudio.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+PackageIdentifier: Twitch.TwitchStudio
+PackageVersion: 0.104.7
+InstallerSwitches:
+  Silent: /silent
+  SilentWithProgress: /silent
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://spotlight-assets.twitchcdn.net/installer/TwitchStudioSetup-network.b0d5a8cf311e73e5e06ada3ea9b6f0c3.exe
+  InstallerSha256: ECF53920563C32BC433CAA4ED9E317691B509D0505435731BE29346B78A1E905
```
Comment (jedieaston) [inline manifests/t/Twitch/TwitchStudio/0.104.7/Twitch.TwitchStudio.installer.yaml:13]:
```suggestion
  InstallerSha256: ECF53920563C32BC433CAA4ED9E317691B509D0505435731BE29346B78A1E905
  ProductCode: {DEE70742-F4E9-44CA-B2B9-EE95DCF372B0}
```

**Review by jedieaston — CHANGES_REQUESTED** [review]

**Comment by jedieaston** [comment]
Hot tip to change the folder name: Press "." on your keyboard when you have GitHub open to this PR in your browser, and a web version of VSCode will launch.
 
 Once it loads, find the files you added, and rename the folder. Then use the source control button on the left to commit (it's automatically pushed).
 
 image

**Comment by sanyer** [comment]
oh, that's sick new addition! Thanks for the tip :-D

btw, what can you say about the fact that version `8.0.0` is not a version of the Twitch apps, but instead is their wrapper's (or something)?
The real version is available only for Twitch Studio, though.

File: manifests/t/Twitch/TwitchStudio/8.0.0/Twitch.TwitchStudio.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Twitch.TwitchStudio
+PackageVersion: 0.104.7
```
Comment (jedieaston) [inline manifests/t/Twitch/TwitchStudio/8.0.0/Twitch.TwitchStudio.installer.yaml:5]:
```suggestion
PackageVersion: 8.0.0
```

**Comment by jedieaston** [comment]
> btw, what can you say about the fact that version 8.0.0 is not a version of the Twitch apps, but instead is their wrapper's (or something)?

I'm not sure why that is (ISVs be crazy :D) but we probably need to use that version number since it's the one in the ARP table so winget can detect it. There will be mitigations for this in schema v1.1 (no eta yet on that though).

**Review by jedieaston — CHANGES_REQUESTED** [review]

**Comment by sanyer** [comment]
> > btw, what can you say about the fact that version 8.0.0 is not a version of the Twitch apps, but instead is their wrapper's (or something)?
> 
> I'm not sure why that is (ISVs be crazy :D) but we probably need to use that version number since it's the one in the ARP table so winget can detect it. There will be mitigations for this in schema v1.1 (no eta yet on that though).

so, v1.1 is not live yet? Links are available at `aka.ms`.

**Comment by jedieaston** [comment]
Schema 1.1 is finalized, but it hasn’t been implemented in the client (the one to many ARP relationship hasn’t been, anyway).

**Comment by sanyer** [comment]
.

> Schema 1.1 is finalized, but it hasn’t been implemented in the client (the one to many ARP relationship hasn’t been, anyway).

oh, ok, got it.
Is there a documentation for the changes?

**Review by jedieaston — APPROVED** [review]

**Comment by jedieaston** [comment]
>  Is there a documentation for the changes?

Not yet. The best I know of is this issue: https://github.com/microsoft/winget-cli/issues/1243.

**Pull request merged**